helpers/scraper.js

```diff
@@ -0,0 +1,366 @@
+    return "Texas";
+  } else if (school.includes("El Paso")) {
+    return "UTEP";
+  } else {
+      return "UT_".concat(usableWords.splice(1, usableWords.length - 1).join("_"));
+  }
+}
+
+const getTeamName = function(school) {
+  let usableWords = [];
+  let splitSchool = school.split(" ");
+  if (school == "Boston University") {
+      return "Boston_U";
+  } if (school.includes("Brigham Young")) {
+      return "BYU";
+  } if (school.includes("California State University") || school.includes("University of California")) {
+      return getCalifornia(school);
+  } if (school.includes("California Polytechnic")) {
+      return "Cal_Poly";
+  } if (school.includes("University of North Carolina")) {
+      return getNC(school);
+  } if (school.includes("University of Wisconsin")) {
+    return getWisco(school);
+  } if (school.includes("University of Texas")) {
+    return getTexas(school);
+  } if (splitSchool[0] == "Pennsylvania") {
+    splitSchool[0] = "Penn";
+  } if (school == "University of Mississippi") {
+    return "Ole_Miss";
+  } if (school == 'Mississippi State University') {
+    return 'Miss State';
+  } if (school == 'Colorado Boulder !University of Colorado Boulder') {
+    return 'Colorado';
+  }
+  for (var i = 0; i < splitSchool.length; i++) {
+      if (splitSchool[i].includes("!")) {
+        return usableWords.join("_");
+      } else {
+        usableWords.push(splitSchool[i]);
+      }
+    } 
+  if (usableWords[usableWords.length - 1] == "University") {
+    usableWords[usableWords.length - 1] = "";
+    usableWords = usableWords.splice(0, usableWords.length - 1);
+    return usableWords.join("_")
+  }
+  return school;
+}
+
+const getSchoolNames = async () => {
+    return new Promise((resolve, reject) => {
+    let names = [];
+    let count = 0;
+    request(
+      'https://en.wikipedia.org/wiki/List_of_NCAA_Division_I_institutions',
+      function(err, resp, body) {
+        if (!err && resp.statusCode == 200) {
+          //console.log('cake');
+          var $ = cheerio.load(body);
+          var count = 0;
+          $('tr', '.sortable').each(function() {
+            const row = $(this)
+              .text()
+              .split('\n');
+            const data = [getTeamName(row[1]), getAbbrev(row[4])];
+            //console.log(data);
+            names.push(data);
+            let splitData = data[0].split("_");
+            //console.log(data);
+            if (splitData[splitData.length - 1] == 'State') {
+              splitData[splitData.length - 1] = 'St';
+              names.push(splitData.join('_'));
+            }
+            if (data[0].split(' ').length > 1 && notTitle(data[0])) {
+              count++;
+            }
+          });
+          resolve(names);
+        } else {
+          console.log(resp.statusCode);
+          reject(err);
+        }
+      }
+    )});
+}
+
+  //console.log(teams);
+  // if you print teams, you'll see that it's empty.
+const parseDate = function(date) {
```

Comment:
Not sure what the point of this is

Reply:
This is because the Date object needs to have years in the form of 20yy whereas the dates in the meets are given as yy